Assignment2/gemawari-GemaGonzalez.rdf

```diff
@@ -1,25 +1,25 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<rdf:RDF xml:base="http://www.ontogrid.net/StickyNote#"
-    xmlns="http://www.ontogrid.net/StickyNote#"
-    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
-    <rdf:Description rdf:about="#Class01">
-	<includes rdf:resource="#Sensor029"/>
-	<includes rdf:resource="#Computer101"/>
-    </rdf:Description>
-    <rdf:Description rdf:about="#Sensor029">
-	<hasMeasurement rdf:resource="#Measurement8401"/>
-    </rdf:Description>
-	<rdf:Description rdf:about="#Measurement8401">
-	<hasTemperature rdf:resource="#29"/>
-	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2010-06-12T12:00:12</atTime>
-    </rdf:Description>
-    <rdf:Description rdf:about="#Computer101">
-	<hasOwner rdf:resource="#User10A"/>
-    </rdf:Description>
-    <rdf:Description rdf:about="#User10A">
-	<hasName rdf:resource="#Pedro"/>
-    </rdf:Description>
-    <rdf:Description rdf:about="#Pedro">
-    </rdf:Description>
-</rdf:RDF>
-
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF xml:base="http://www.ontogrid.net/StickyNote#"
+    xmlns="http://www.ontogrid.net/StickyNote#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <rdf:Description rdf:about="#Class01">
+	<includes rdf:resource="#Sensor029"/>
+	<includes rdf:resource="#Computer101"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+	<hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+	<rdf:Description rdf:about="#Measurement8401">
+	<hasTemperature rdf:resource="#29"/>
+	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2010-06-12T12:00:12</atTime>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Computer101">
+	<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#User10A">
+	<hasName rdf:resource="#Pedro"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Pedro">
```

Comment:
Check other student's submission. This is wrong as this should be a literal

Reply:
Ahh, I see that this is from anotehr student.

Reply:
fixed